voting/ethereum/instructions/guardian_registration.md

```diff
@@ -0,0 +1,81 @@
+## Guardian Registration
+In order to participate in the voting, a Guardians must first enroll. 
+An Orbs Guardian is identified and delegated to by its Ethereum address. 
```

Comment:
"its Ethereum address"? they are supposed to be people... consider "her" instead of "its". or change to plural form (Orbs Guardians) with "their"...?

Reply:
fixed